### ISSUE TITLE
a11y instrumentation of VerticalCheckBoxGroup

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#349]
... which should be similar to VerticalAquaRadioButtonGroup. See https://github.com/phetsims/sun/issues/344#issuecomment-377401457.

**Comment by marlitas** [comment]
Meeting 11/5
- This is part of https://github.com/phetsims/sun/issues/775
- Assigning @jessegreenberg for next steps.
